examples/navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import 'rxjs';
```

Comment:
This is originally why I added it – the operators I wanted were not on the `ActionsObservable` and I was immediately sure which operators I was going to want to use. However, I just removed it and now *Just Works*.